lib/dnd/monterey-macos-dnd.mm

```diff
@@ -10,6 +10,7 @@ + (bool)isEnabled {
 
   // 1. check assertion
   bool isDNDEnabled = [self enabledByAssertion];
```

Comment:
looks like my initial naming not accurate - maybe rename it to "isFocusModeEnabled"